### PR TITLE
🌐 Add Chinese translation for `docs/advanced/response-cookies.md`

### PR DESCRIPTION
**Pull request by jaystone776**
As title

### CODE REVIEW

File: docs/zh/docs/advanced/response-cookies.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# 响应 - Cookie
+
+## 使用 `Response` 参数
+
+*路径操作函数* 中可以声明 `Response` 类型的参数。
+
+现在，在*临时*响应对象中设置 cookie。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-cookies.md:7]:
```suggestion
然后你可以在这个*临时*的响应对象中设置 cookie。
```

File: docs/zh/docs/advanced/response-cookies.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+然后，在响应内设置 Cookie，并返回响应：
+
+```Python hl_lines="10-12"
+{!../../../docs_src/response_cookies/tutorial001.py!}
+```
+
+!!! tip "提示"
+
+    注意，不使用 `Response` 参数直接返回响应时，FastAPI 直接返回响应。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-cookies.md:35]:
```suggestion
    注意，如果不使用 `Response` 参数而是直接返回一个响应对象，FastAPI 会将该响应不加处理直接返回给客户端。
```

File: docs/zh/docs/advanced/response-cookies.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    
+    因此，必须确保数据类型是正确的。例如，使用 `JSONResponse` 返回响应时，数据类型要兼容 JSON。
+    
+    而且不能发送经 `response_model` 筛选过的数据。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-cookies.md:39]:
```suggestion
    而且不能发送需要通过 `response_model` 进行校验过滤的数据。
```

File: docs/zh/docs/advanced/response-cookies.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+!!! note "技术细节"
+
+    您也可以使用 `from starlette.responses import Response` 或 `from starlette.responses import JSONResponse`。
+    
+    **FastAPI** 的 `fastapi.responses` 与 `starlette.responses` 一样，只是为开发者提供的快捷方式，但其中绝大部分可用的响应都直接继承自 Starlette。
+    
+    `Response` 常用于设置响应头或 Cookie，因此，**FastAPI** 在 `fastapi.Response` 中提供了支持。
+
+所有可用参数及选项，详见 <a href="https://www.starlette.io/responses/#set-cookie" class="external-link" target="_blank">Starlette 官档</a>。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-cookies.md:51]:
```suggestion
所有可用参数及选项，详见 <a href="https://www.starlette.io/responses/#set-cookie" class="external-link" target="_blank">Starlette 官方文档</a>。
```

**Review by waynerv — CHANGES_REQUESTED** [review]

**Review by waynerv — APPROVED** [review]

**Comment by tiangolo** [comment]
It seems this was translated at some point in another PR. So I'll close this one, but thanks for the effort! ☕

**Pull request closed without merging**